pkg/inventory/model/client.go

```diff
@@ -127,7 +111,6 @@ func (r *Client) Close(purge bool) error {
 			"path",
 			r.path)
 	}
-	r.db = nil
```

Comment:
The `Client` is accessed concurrently.  Setting the sqlite DB (field) to nil ^^ can result in SEGV.  I observed this while testing the frequent open/close cycles.  The other refactoring of splitting out the creating of the schema into a separate build() method is leftover from when (at one point), I added a _finalizer_.  I reverted the finalizer but thought this was a healthy refactor and left it.